backend/sprocs.py

```diff
@@ -19,13 +19,15 @@ def timer(func: object):
     Args:
         func (object): A function object
     """
+
```

Comment:
While not explicitly defined in PEP, the remainder of the project does not include a line of whitespace between a docstring and the code.  I think it would be better to remove the extra whitespace line behind the docstring.  I'm indifferent about separating the two return calls.

Reply:
yep, i'm fine with that. My formatter automatically creates a whitespace here, which is probably why it was added oops.